core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala

```diff
@@ -34,11 +34,21 @@ private[ui] class PoolTable(pools: Map[Schedulable, PoolData], parent: StagesTab
     <table class="table table-bordered table-striped table-condensed sortable table-fixed">
       <thead>
         <th>Pool Name</th>
-        <th>Minimum Share</th>
-        <th>Pool Weight</th>
+        <th>
+          <span data-toggle="tooltip" data-placement="top" title="Pools can be set a minimum share
+           of CPU cores to allocate">Minimum Share</span>
+        </th>
+        <th>
+          <span title="This controls the pool's share of the cluster relative to other pools"
```

Comment:
share of what? "Pool's share of cluster resources relative to others"

Reply:
ok i will update the tooltips to this "Pool's share of cluster resources relative to others"